jenkins/milestone-deploy/Jenkinsfile

```diff
@@ -0,0 +1,61 @@
+pipeline {
+  agent any
+
+  options {
+    buildDiscarder(logRotator(numToKeepStr:'15'))
+    disableConcurrentBuilds()
+    timeout(time: 120, unit: 'MINUTES')
+  }
+
+  parameters {
+    choice(name: 'RELEASE_TYPE', choices: ['M0', 'M1', 'M2', 'M3', 'RC1', 'RC2', 'GA'], description:
+      '''
+        Kind of milestone release to build.
+      ''')
+  }
+
+  environment {
+    DOWNLOAD_AREA = '/home/data/httpd/download.eclipse.org/modeling/tmf/xtext'
+    REPOSITORY_PATH="${DOWNLOAD_AREA}/updates/milestones"
+  }
+
+  tools {
+    maven "apache-maven-3.8.6"
```

Comment:
is it intentional not to have the latest 3.8 version here?

Reply:
as I said in other PRs, that's what I found in the original Jenkinsfile, including the one we use for standard CI. I didn't dare to touch that for the moment. If we want to go with 3.8, maybe we should do that first in the standard Jenkinsfile so that we won't have bad surprise upon release.